src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -768,6 +797,14 @@ protected override void SerializeCore(XmlElement element)
                 }
             }
 
+            internal override void TrackAnalytics()
+            {
+                Dynamo.Logging.Analytics.TrackEvent(
+                    Logging.Actions.Create,
+                    Logging.Categories.NodeOperations,
+                    Node.Name);
```

Comment:
I am not sure if Node.Name is not always populated and has value. You can either use NickName or the extension method OriginalName().